modules/tinymce/src/themes/silver/test/ts/webdriver/editor/DisabledMenuItemTest.ts

```diff
@@ -0,0 +1,55 @@
+import { Mouse, RealKeys, UiFinder, Waiter } from '@ephox/agar';
+import { before, describe, it } from '@ephox/bedrock-client';
+import { McEditor, TinyUiActions } from '@ephox/mcagar';
+import { SugarBody } from '@ephox/sugar';
+
+import TablePlugin from 'tinymce/plugins/table/Plugin';
+import Theme from 'tinymce/themes/silver/Theme';
+
+describe('webdriver.tinymce.themes.silver.editor.menubar.DisabledMenuItemTest', () => {
+
+  before(() => {
+    Theme();
+    TablePlugin();
+  });
+
+  const settings = {
+    base_url: '/project/tinymce/js/tinymce',
+    plugins: 'table',
+    menu: {
+      table: { title: 'Table', items: 'inserttable | cell row column | tableclass tablecellclass tablecellvalign tablecellborderwidth tablecellborderstyle tablecaption tablecellbackgroundcolor tablecellbordercolor | advtablesort | tableprops deletetable' }
+    },
+    menubar: 'table',
+  };
+
+  const pOpenTableMenu = (editor) => {
+    const tableMenuItemSelector = '[role="menuitem"]:contains("Table")';
+    TinyUiActions.clickOnMenu(editor, tableMenuItemSelector);
+    return Waiter.pTryUntil('Wait for a specific menu to open', () => UiFinder.exists(SugarBody.body(), tableMenuItemSelector));
```

Comment:
This will always be present as you're using the same selector to open the menu.

That is, `tableMenuItemSelector` is always in the menubar. You need to use a selector to look for the vertical align button for this assertion instead.

`const verticalAlignMenuItemSelector = 'div[title="Vertical align"]';`

I suspect this is the root cause of your issues

Reply:
I'm also not sure this label makes sense? Should it be something like "Wait for table menu to open"?